src/annotator/pdf-sidebar.js

```diff
@@ -12,8 +16,98 @@ const defaultConfig = {
   },
 };
 
+// The viewport and controls for PDF.js start breaking down below about 670px
+// of available space, so only render PDF and sidebar side-by-side if there
+// is enough room. Otherwise, allow sidebar to overlap PDF
+const MIN_PDF_WIDTH = 680;
+
 export default class PdfSidebar extends Sidebar {
   constructor(element, config) {
     super(element, { ...defaultConfig, ...config });
+
+    this.lastSidebarLayoutState = {
+      expanded: false,
+      width: 0,
+      height: 0,
+    };
+
+    this.window = /** @type {HypothesisWindow} */ (window);
+    this.pdfViewer = this.window.PDFViewerApplication?.pdfViewer;
+    this.pdfContainer = this.window.PDFViewerApplication?.appConfig?.appContainer;
+
+    // Prefer to lay out the sidebar and the PDF side-by-side (with the sidebar
+    // not overlapping the PDF) when space allows
+    this.sideBySide =
```

Comment:
It doesn't look like this field is currently used outside of this method. Could just be a local var?

Reply:
Ah, legacy of refactor. Sure.

Reply:
After reflection, I've left this in place. It feels like a meaningful property on the `PdfSidebar` and makes subsequent logic more concise. It's kind of half dozen of one, six of the other...